src/cmd_processor.cc

```diff
@@ -0,0 +1,46 @@
+#include "cmd_processor.h"
+
+#include <libqalculate/Calculator.h>
+
+namespace bb
+{
+    class CMDProcessor::Impl
+    {
+    public:
+        Impl();
+        std::string process(const std::string &cmd);
+
+        Calculator *calc;
+    };
+}
+
+// Internal implementation
+
+bb::CMDProcessor::Impl::Impl()
+    : calc(new Calculator())
+{
+    calc->useIntervalArithmetic(false);
+    calc->loadExchangeRates();
+    calc->loadGlobalCurrencies();
+    calc->loadGlobalDefinitions();
+    calc->loadLocalDefinitions();
+}
+
+std::string bb::CMDProcessor::Impl::process(const std::string &cmd)
+{
+    return calc->calculateAndPrint(calc->unlocalizeExpression(cmd));
```

Comment:
Use unlocalizeExpression affects the argument separators and decimal signs. We should use the system configuration or maybe a special setting provided by the user